application-src/amstar.c

```diff
@@ -619,7 +619,9 @@ static char *validate_command_options(
     for (copt = argument->command_options; copt != NULL; copt = copt->next) {
 	char *opt = (char *)copt->data;
 
-	if (g_str_has_prefix(opt, "--compress-command")) {
+	if (g_str_has_prefix(opt, "new-volume-script") ||
+		g_str_has_prefix(opt, "-new-volume-script") ||
+		g_str_has_prefix(opt, "--new-volume-script")) {
 	   return opt;
```

Comment:
This seems to still be a blacklist.  I assume that amstar, like amgtar, accepts unique prefixes of options, so this approach will not prevent, for example, `--new-volume-scr`.

Reply:
Hello Dustin,
Thank you for your review comment.
Yes, we are working on WHITELIST approach for amstar application.